src/main/java/com/google/sps/data/AccessSecret.java

```diff
@@ -0,0 +1,44 @@
+package com.google.sps.data;
+
+import com.google.cloud.secretmanager.v1.AccessSecretVersionResponse;
+import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1.SecretVersionName;
+import java.io.IOException;
+import java.lang.Throwable;
+
+public class AccessSecret {
+  private String key = null;
+  private static AccessSecret instance = new AccessSecret();
+
+  private AccessSecret(){}
+
+  private String accessSecretVersion() {
+    String projectId = "team-38-step-2020";
+    String secretId = "API_KEY";
+    String versionId = "2";
```

Comment:
these should be Final since you won't be changing them

Reply:
done